lib/new_relic/agent/vm/mri_vm.rb

```diff
@@ -49,7 +49,19 @@ def gather_ruby_vm_stats(snap)
           end
 
           if supports?(:constant_cache_invalidations)
-            snap.constant_cache_invalidations = RubyVM.stat[:global_constant_state]
+            snap.constant_cache_invalidations = gather_constant_cache_invalidatiions
+          end
+        end
+
+        def gather_constant_cache_invalidatiions
```

Comment:
small typo: 
```suggestion
        def gather_constant_cache_invalidations
```

Reply:
So my hardware apparently let me down with a bit of key chatter and then my own laziness in copy and pasting and not proofreading let me down further! Thanks for having my back!

Reply:
> Would you mind creating an issue

Good call. I've gone ahead and updating the original issue and marked this PR as not resolving it so that it can continue to track efforts to enhance our reporting with 3.2 specific functionality.